code/drasil-example/Drasil/GamePhysics/DataDefs.hs

```diff
@@ -351,6 +356,22 @@ kEnergyDesc :: Sentence
 kEnergyDesc = foldlSent [S "The", phrase QP.kEnergy,
  S "of an object is the", phrase QP.energy,
  S "it possess due to its motion"]
+-----------------------DD16 Moment Of Inertia--------------------------------------------------------
+
+momentOfInertiaDD :: DataDefinition
+momentOfInertiaDD = ddNoRefs momentOfInertia [{-- Derivation --}] "momentOfInertia"
+ [momentOfInertiaDesc, makeRef2S assumpOT] 
+
+momentOfInertia :: QDefinition
+momentOfInertia = mkQuantDef QP.momentOfInertia momentOfInertiaEqn
+
+momentOfInertiaEqn :: Expr
+momentOfInertiaEqn = (sumAll (Atomic "i") (sy massI) * (sy rRot $^ 2))
```

Comment:
Those outer () are still causing `hLint` to be unhappy.

Reply:
HLint suggestion as you said changes the meaning, but I took out the outer brackets.
<img width="526" alt="image" src="https://user-images.githubusercontent.com/43192745/59400564-cd794180-8d65-11e9-8864-7be09112dcd0.png">

Reply:
@JacquesCarette , if I don't use the HLint suggestion for any reason, is the build always going to fail?

Reply:
Probably.  I think the correct expression here should be
```haskell
sumAll (Atomic "i") $ sy massI * (sy rRot $^2)
```

Reply:
(but I have not tested it)

Reply:
Okay so why did I not try this earlier.... I thought about it but did not try it out thinking  the "()" is as good as "$". So HLint does not like brackets.

Reply:
I will push and see if this works

Reply:
Yay, it worked. Thanks

Reply:
Note that you should be able to test this without pushing - I do believe that 'make' runs hlint -- right @Mornix ?

Reply:
Usually when my local build is successful I still get Hlint errors if the build fails.

Reply:
We should definitely make sure that hlint runs locally too.